### PR TITLE
feat(LinuxFramebuffer): Allow set DRM output Connector 

### PR DESCRIPTION
**Pull request by workgroupengineering**
## What does the pull request do?

Allow set DRM output Connector 

## What is the current behavior?

Always use the predefined output connector.

## Checklist

- [ ] Added unit tests (if possible)?
- [x] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/avalonia-docs with user documentation

### CODE REVIEW

**Review by kekekeks — COMMENTED** [review]
LGTM in general, but rather noisy on the whitespace changes

**Comment by workgroupengineering** [comment]
If you want I delete the commit of the code formatting 444af273914366abac71893cf3ab35735b50ddc6

File: src/Linux/Avalonia.LinuxFramebuffer/EnumerableExtesions.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Avalonia.LinuxFramebuffer;
+
+internal static class EnumerableExtesions
+{
+    public static IEnumerable<TSource> Where<TSource, TArg>(this IEnumerable<TSource> source, Func<TSource, TArg, bool> predicate, TArg arg)
```
Comment (maxkatz6) [inline src/Linux/Avalonia.LinuxFramebuffer/EnumerableExtesions.cs:8]: Do we really need this extension here? Simple one pass foreach can work there just fine.

Comment (workgroupengineering) [reply src/Linux/Avalonia.LinuxFramebuffer/EnumerableExtesions.cs:8]: Because I would like to avoid closure allocation

Comment (maxkatz6) [reply src/Linux/Avalonia.LinuxFramebuffer/EnumerableExtesions.cs:8]: Isn't this code executed once? Either way, with foreach the same closure can be avoided as well.

Comment (maxkatz6) [reply src/Linux/Avalonia.LinuxFramebuffer/EnumerableExtesions.cs:8]: Note, in Linq Where operator has number of optimizations depending on the input collection type. https://github.com/dotnet/runtime/blob/main/src/libraries/System.Linq/src/System/Linq/Where.cs#L36C28-L36C46

**Review by maxkatz6 — APPROVED** [review]

**Pull request merged**